docs/containers.conf.5.md

```diff
@@ -0,0 +1,347 @@
+% containers.conf(5) Container engine configuration file
+
+# NAME
+containers.conf - The container engine configuration file specifies default
+configuration options and command-line flags for container engines.
+
+# DESCRIPTION
+Container engines like Podman & Buildah read containers.conf file, if it exists
+and modify the defaults for running containers on the host. containers.conf uses
+a TOML format that can be easily modified and versioned.
+
+Container engines read the /usr/share/containers/containers.conf and
+/etc/containers/containers.conf files if they exists.  When running in rootless
+mode, they also read $HOME/.config/containers/containers.conf files.
+
+Fields specified in containers conf override the default options, as well as
+options in previously read containers.conf files.
+
+Note not all options are supported in all container engines.
+
+# FORMAT
+The [TOML format][toml] is used as the encoding of the configuration file.
+Every option is nested under its table. No bare options are used. The format of
+TOML can be simplified to:
+
+    [table1]
+    option = value
+
+    [table2]
+    option = value
+
+    [table3]
+    option = value
+
+    [table3.subtable1]
+    option = value
+
+## CONTAINERS TABLE
+The containers table contains settings pertaining to the OCI runtime that can
+configure and manage the OCI runtime.
+
+**additional_devices**=[]
+  List of additional devices.
+  Specified as 'device-on-host:device-on-container:permissions',
+  for example: "--additional-devices=/dev/sdc:/dev/xvdc:rwm".
```

Comment:
I am still VERY not in favor of passing these as strings. Guaranteeing identical parsing between Podman and Buildah is nontrivial.

Reply:
They should be using the same library functions to parse them.